quizzer/src/Answer.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default class Answer extends React.Component {
+  render() {
+    let {title, score} = {...this.props.data};
+    return (
+      <label >
+        <input
+          className='answer'
+          type='checkbox'
```

Comment:
We likely only want to allow users to select a single answer per question. The radio element can take a `name` attribute that will limit our selection. See example [here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_radio).

The name should be unique per question. i.e. all of the answer elements in question one might have a `name` attribute equal to `answer-group-1`.